ably/rest_client.go

```diff
@@ -66,21 +62,19 @@ func (c *RestClient) Time() (time.Time, error) {
 		return time.Time{}, err
 	}
 	if len(times) != 1 {
-		return time.Time{}, fmt.Errorf("Expected 1 timestamp, got %d", len(times))
+		return time.Time{}, fmt.Errorf("expected 1 timestamp, got %d", len(times))
```

Comment:
I'll change all text messages to have lower-case names a bit latter. This is how idiomatic Go error messages are.